airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -216,11 +217,14 @@ def run_query(
         :type udf_config: list
         :param use_legacy_sql: Whether to use legacy SQL (true) or standard SQL (false).
         :type use_legacy_sql: boolean
+        :param maximum_billing_tier: Positive integer that serves as a multiplier of the basic price.
+        :type maximum_billing_tier: integer
         """
         configuration = {
             'query': {
                 'query': bql,
-                'useLegacySql': use_legacy_sql
+                'useLegacySql': use_legacy_sql,
+                'maximumBillingTier': maximum_billing_tier
```

Comment:
Hi Chris, I'will test it tomorrow and let you know. I did not got the chance today to properly test it. Although I made the changes this morning and all my jobs ran without error. I think None works just fine, but I will test it a bit more tomorrow.